main.go

```diff
@@ -14,12 +14,20 @@ import (
 	"path/filepath"
 	"syscall"
 
+	"cloud.google.com/go/storage"
 	"github.com/fsouza/fake-gcs-server/fakestorage"
 	"github.com/fsouza/fake-gcs-server/internal/checksum"
 	"github.com/fsouza/fake-gcs-server/internal/config"
 	"github.com/sirupsen/logrus"
 )
 
+var DefaultACL = []storage.ACLRule{
```

Comment:
This doesn't really need to be a global, let's inline it (I know it means we'll repeat it in tests, but that's OK).